rules/detect-buffer-noassert.js

```diff
@@ -11,59 +11,67 @@
 
 const names = [];
 
-module.exports = function(context) {
-
-  const read = [
-    'readUInt8',
-    'readUInt16LE',
-    'readUInt16BE',
-    'readUInt32LE',
-    'readUInt32BE',
-    'readInt8',
-    'readInt16LE',
-    'readInt16BE',
-    'readInt32LE',
-    'readInt32BE',
-    'readFloatLE',
-    'readFloatBE',
-    'readDoubleL',
-    'readDoubleBE'
-  ];
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Detect calls to "buffer" with `noAssert` flag set. From the Node.js API docs: "Setting `noAssert` to true skips validation of the `offset`. This allows the `offset` to be beyond the end of the `Buffer`."',
```

Comment:
This should be just one sentence and should not contain markdown.

Reply:
I was unaware of this guideline. I will do another iteration of all the rule meta tags, and commit to this PR.